Makefile

```diff
@@ -244,3 +244,15 @@ channels-gocode:
 
 examples:
 	go install k8s.io/kops/examples/kops-api-example/...
+
+# -----------------------------------------------------
+# api machinery regenerate
+
+apimachinery:
+	#go install ./cmd/libs/go2idl/conversion-gen
+	~/k8s/bin/conversion-gen  --input-dirs k8s.io/kops/pkg/apis/kops/v1alpha1 --v=8  --output-file-base=zz_generated.conversion
```

Comment:
Well... so the challenge is that the fix I had to make it k8s so the conversion generator works only just landed today.  So we also have to update k8s to the latest, or pull it out into the shared repo.  I'm inclined to do the latter.  It's a placeholder for actually installing the code for real